gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# WIP: Mid-Sprint PR

### Description

Pulling mid-sprint stuff into `master`. Basically just fixing the popup bug.

### Testing Directions

```
./setup
python manage.py runserver
```

1.  Make matches and see that the popup works as it should.

### Checklist
